lib/spack/spack/cmd/info.py

```diff
@@ -71,6 +71,10 @@ def variant(s):
     return spack.spec.enabled_variant_color + s + plain_format
 
 
+def license(s):
+    return spack.spec.version_color + s + plain_format
```

Comment:
```suggestion
    return spack.spec.VERSION_COLOR + s + plain_format
```

This changed on `develop`, which is why CI is failing.

Reply:
Rebased and fixed. Thanks for suggesting the fix!